### PR TITLE
update 2.0.2

### PR DESCRIPTION
**Pull request by rvalieris**

### CODE REVIEW

**Comment by jdblischak** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,24 +25,41 @@ build:
...
 
 about:
-  home: https://github.com/MangoTheCat/callr
+  home: https://cran.r-project.org/package=callr
+  dev_url: https://github.com/r-lib/callr
   license: MIT
   summary: It is sometimes useful to perform a computation in a separate R process, without affecting
     the current R process at all. This packages does exactly that.
   license_family: MIT
+  license_file: LICENSE
```
Comment (jdblischak) [inline recipe/meta.yaml]: @rvalieris Could you please remove the `license_file` entry. The file `LICENSE` shipped with R packages licensed under MIT is just a stub, not a full license: https://cran.r-project.org/web/packages/callr/LICENSE

Comment (rvalieris) [reply recipe/meta.yaml]: good catch, didnt notice that.

**Comment by rvalieris** [comment]
ping !

**Pull request merged**